readme.md

```diff
@@ -30,7 +30,7 @@ when no error occurred) and secondly the data generated by that call.
 | `.addConfig(key, value[, handlerFn])` |  add a local configuration property |
 | `.addRemote(name, repo, handlerFn)` | adds a new named remote to be tracked as `name` at the path `repo` |
 | `.addTag(name, handlerFn)` | adds a lightweight tag to the head of the current branch |
-| `.branch([options, ][handlerFn])` | gets a list of all branches, calls `handlerFn` with two arguments, an error object and [BranchSummary](src/BranchSummary.js) instance. When supplied, the options can be either an array of arguments supported by the [branch](https://git-scm.com/docs/git-branch) command or a standard [options](#how-to-specify-options) object. |
+| `.branch([options, ][handlerFn])` | gets a list of all branches, calls `handlerFn` with two arguments, an error object and [BranchSummary](src/BranchSummary.js) instance. The options-parameter is not necessary. When supplied, the options can be either an array of arguments supported by the [branch](https://git-scm.com/docs/git-branch) command or a standard [options](#how-to-specify-options) object. |
```

Comment:
IMO *optional* is the best term in this case and would make the sentence even clearer.